opus/application/static_media/js/browse.js

```diff
@@ -2729,15 +2737,14 @@ var o_browse = {
         let tab = opus.getViewTab();
         let targetElem = e.currentTarget;
         let toolElem = `${tab} .op-edit-field-tool`;
-        let left = $(targetElem).position().left +
-                    $(`${tab} .op-data-table-view .table`).offset().left;
+        let left = $(targetElem).position().left;
         let top = $(targetElem).offset().top +
                   $(targetElem).outerHeight();
 
         if (tab === "#cart") {
-            if ($(".cart_details:visible").length > 0) {
+            if ($(".op-cart-details:visible").length > 0) {
                 // need to calculate a new offset if the table has scrolled left behind the details window
-                left -= $(".cart_details").outerWidth() + $(".cart_details").offset().left;
+                //left -= $(".op-cart-details").outerWidth() + $(".op-cart-details").offset().left;
```

Comment:
This if statement block can be removed since the code is commented out.

Reply:
It doesn't look like this comment was ever acted on